server/server.go

```diff
@@ -821,6 +827,11 @@ func (s *Server) GetServiceMiddlewareConfig() *config.ServiceMiddlewareConfig {
 	return cfg
 }
 
+// SetEnableLocalTSO sets enable-local-tso flag of Server .
```

Comment:
Please add a comment to tell that this function should only be used in the test.